app/routeConfig.js

```diff
@@ -2,15 +2,15 @@ import NotFound from '@containers/NotFoundPage/Loadable';
 import HomeContainer from '@containers/HomeContainer/Loadable';
 import routeConstants from '@utils/routeConstants';
 import ArtistsContainer from '@containers/ArtistsContainer/index';
-import TrackDetails from './components/TrackDetails/index';
+import TrackDetailsContainer from './containers/TrackDetailsContainer/index';
```

Comment:
Use relative import only if the file is in same directory otherwise use absolute import. Update them everywhere